neptune/new/attributes/series/string_series.py

```diff
@@ -53,3 +53,29 @@ def get_last(self, wait=True) -> str:
             self._experiment.wait()
         val = self._backend.get_string_series_attribute(self._experiment_uuid, self._path)
         return val.last
+
+    def fetch_values(self, include_timestamp=True):
+        # pylint: disable=import-outside-toplevel
+        import pandas as pd
```

Comment:
Is there any good reason for implementing this inside funciton?

If so, maybe we should mention it in comment.

Reply:
In all of out project we do it like this - this kind of import means we do not initialize-on-import pandas lib (I think that was the reasoning behind it). At some point in time, `import neptune` took a long time because all the libs got initialized on import

Reply:
Makes sense